docs/changelog.md

```diff
@@ -1,6 +1,7 @@
 dev
 
 - Added reinstall command for reinstalling a single venv.
+- [bugfix] fix bug with reinstall-all. Now the initial specifier is used 
```

Comment:
Can you change this to past tense and make full sentences, i.e. "Fixed bug with reinstall-all."

Reply:
Done